src/TorchSharp/TorchVision/Ops.cs

```diff
@@ -1,6 +1,9 @@
 // Copyright (c) .NET Foundation and Contributors.  All Rights Reserved.  See LICENSE in the project root for license information.
 
 using static TorchSharp.torch;
+using System.Range;
```

Comment:
Not available on all versions of .NET that we are targeting.

Reply:
Removed the unsupported dependencies.